build/Codegen.cmake

```diff
@@ -220,3 +220,113 @@ function(merge_yaml)
     WORKING_DIRECTORY ${EXECUTORCH_ROOT}
   )
 endfunction()
+
+function(append_filelist name outputvar)
```

Comment:
Please add a comment explaining what this does and what it expects for (and will do to) `outputvar`